### PR TITLE
Google sheets

### LINKED ISSUE #142: Refactor Metadata management
### Action
Change how metadata is loaded into the report from being stored in the preprocessing scripts to being pulled from a google sheet. 

### Context
Managing the metadata in each file was getting out of hand and not scaling well.  Using a common sheet and function will reduce maintenance effort and cut down on errors.

### Effort
8? 13? Lots of grunt work.  

### Things to do:

- [x] Add additional columns to the metadata sheet: reference URL, internal url, constraints, etc.
- [x] Synchronize sheet with report including values not aligning (Secure vs none)  
- [x] Add some missing rows (ws_rr, rivers_rr) 
- [x] Make decision on how to handle dates
- [x] Add function to run all the preprocessing scripts in one go

### PR DESCRIPTION
**Pull request by PJCoates**
Added Google sheets metadata to all records. resolve #142

### CODE REVIEW

File: reports/sections/whelk/whelk_preprocessing.R [hunk 1]
```diff
@@ -11,21 +11,22 @@ source(here::here("config.R"))
...
 loadResult <- load_rdata(c("CommonData", "whelk_rr"), regionStr)
 
-# ---------------------WHELK SURVEY-----------------------------------
+print("---------------------WHELK SURVEY----------------------------------- ")
+if (globalControlEnv$updateGeoms) {
+
 # Open Data dataset
-pkgId <- "e606ad01-843f-4730-93fa-f944dfbbbf07"
+whelkpkgId <- "e606ad01-843f-4730-93fa-f944dfbbbf07"
 resId <- "5355ea5b-ea37-4a4b-8f61-9d6b716b7d17"
```
Comment (stoyelq) [inline reports/sections/whelk/whelk_preprocessing.R:19]: The resID is no longer used, so you can remove this line.

File: reports/sections/wsdb/wsdb_preprocessing.R [hunk 2]
```diff
@@ -34,19 +36,10 @@ wsdb_sf <- merge(wsdb_sf, cetLegend, by='Scientific Name')
 wsdb_sf <- dplyr::select(wsdb_sf, CNAME, 'Scientific Name', YEAR, Legend, LATITUDE, LONGITUDE)
 wsdb_sf <- sf::st_crop(wsdb_sf, region_sf)
 
-wsdb_rr <- list("title" = "Whale Sightings Database",
```
Comment (stoyelq) [inline reports/sections/wsdb/wsdb_preprocessing.R:37]: Get google metadata only gets the metadata part of the RR, we still need to keep the initalization and set the title, attribute, and data_sf field.  Take a look at permits_preprocessing.R for an example of what this looks like with the google sheet.

**Comment by stoyelq** [comment]
I added a couple of small comments, beyond that it looks great!

File: reports/sections/whelk/whelk_preprocessing.R (hunk 1 above)
Comment (PJCoates) [reply reports/sections/whelk/whelk_preprocessing.R]: resID removed and change pushed

**Pull request merged**